hi-ml-histopathology/src/histopathology/datasets/default_paths.py

```diff
@@ -4,7 +4,7 @@
 #  ------------------------------------------------------------------------------------------
 
 PANDA_DATASET_ID = "PANDA"
-PANDA_TILES_DATASET_ID = "PANDA_tiles"
+PANDA_TILES_DATASET_ID = "PANDA_tiles_20220427"
```

Comment:
Would it make sense to remove the hard-coded path from everywhere else and always read from here instead?

Reply:
Yes, this is a good idea. I have now removed it from everywhere else except here